src/components/grid-view/GridView.js

```diff
@@ -101,7 +102,7 @@ class GridView extends React.Component<Props> {
                 width={width}
                 gridClassName="bdl-GridView-body"
                 rowClassName="bdl-GridView-tableRow"
-                scrollToIndex={0}
+                scrollToIndex={scrollToRow || 0}
```

Comment:
The scrollToIndex refers to the index of the row, yes. I think we should keep the name scrollToRow for the GridView component, because it's more clear